docs/pages/repo/docs/handbook/linting/eslint.mdx

```diff
@@ -99,11 +142,15 @@ We can then import the config like this:
 ```js filename="apps/web/.eslintrc.js"
 module.exports = {
   root: true,
-  extends: ["custom"],
+  extends: ["custom/next"],
 };
 ```
 
-By adding `custom` to our `extends` array, we're telling ESLint to look for a package called `eslint-config-custom` - and it finds our workspace.
+By adding `custom/next` to our `extends` array, we're telling ESLint to look for a package called `eslint-config-custom`, and reference the file `next.js`.
+
+<Callout type="info">
+  You can avoid specifing the file by setting the entrypoint for your package as the file to load. For example, setting `main: 'next.js'` in the `package.json`, could be loaded as simply `extends: ["custom"]` in your `.eslintrc.js`.
```

Comment:
> You can avoid specifing the file by setting the entrypoint for your package as the file to load

This sentence feels a little confusing to me, but I'm not sure how to fix it. The following sentence makes it crystal clear so I feel like maybe keeping the first one brief would help.

I think that "You can avoid specifing the file by setting the entrypoint for your package." followed by the example is clear enough.

Reply:
Good call - updated here: https://github.com/vercel/turbo/pull/5831